packages/flutter/test/material/dialog_test.dart

```diff
@@ -2766,9 +2766,10 @@ void main() {
     expect(find.text('Dialog2'), findsOneWidget);
   });
 
-  testWidgets('Uses open focus traversal when overridden', (WidgetTester tester) async {
+  testWidgetsWithLeakTracking('Uses open focus traversal when overridden', (WidgetTester tester) async {
     final FocusNode okNode = FocusNode();
     final FocusNode cancelNode = FocusNode();
+    addTearDown(() { okNode.dispose(); cancelNode.dispose(); });
```

Comment:
nit formatting: place statements on their own line. Same for other occurrences in this pr.

Reply:
Done, but how about exception for OverlayEntry? With right formatting it will take four lines, that will make it harder for reader to focus on test functionality.

I tried this, but the test engine swapped the order of operations and failed:

```
    addTearDown(() => entry.remove());
    addTearDown(() => entry.dispose());
```

Reply:
Would `addTearDown(entry..remove..dispose);` work? If not it is okay as is.